src/main/java/io/cdap/e2e/pages/actions/CdfPipelineRunAction.java

```diff
@@ -119,6 +128,14 @@ public static void waitTillPipelineRunCompletes(long timeoutInSeconds) {
     ));
   }
 
+  /**
+   * Stop the Pipeline
+   */
+  public static void stopPipeline() {
+    ElementHelper.clickOnElement(CdfPipelineRunLocators.stop);
+    WaitHelper.waitForElementToBeDisplayed(CdfPipelineRunLocators.stoppedStatus);
```

Comment:
Similarly here, transition to stopped state will not be instantaneous always.

Reply:
This wait step will wait till the Pipeline is transitioned to stopped state with the default timeout (60 s). Also added an overloaded method which expects a Timeout.

Reply:
And then use this step to verify that pipeline status is `stopped`.
https://github.com/cdapio/cdap-e2e-tests/blob/develop/src/main/java/stepsdesign/PipelineSteps.java#L358

Reply:
Removed the line waiting for the status to transition to Stopped.